src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmMissingResponseCleanupImpl.java

```diff
@@ -36,7 +36,7 @@ public ArmMissingResponseCleanupImpl(ExternalObjectDirectoryRepository externalO
                                          ArmResponseFileHelper armResponseFileHelper) {
         super(externalObjectDirectoryRepository, objectRecordStatusRepository, externalLocationTypeRepository, armDataManagementApi, userIdentity,
               batchCleanupConfiguration, armDataManagementConfiguration, currentTimeHelper, armResponseFileHelper,
-              "ArmMissingResponseCleanup");
+              "", "ArmMissingResponseCleanup");
```

Comment:
By Passing a empty prefix, will not fail the validation in ArmResponseFileHelper.getCorrespondingArmFilesForManifestFilename? Cleanup logic during ARM Missing response replay flow should be independent of DETS or DARTS i.e. any manifest files corresponding to the EOD should get deleted irrespective of whether its DETS or DARTs.

Reply:
Lets have a chat around this :).